src/utilities/routerUtilities.js

```diff
@@ -0,0 +1,18 @@
+function handleServiceError(error, res) {
+    console.error(error);
+
+    let statusCode = error.name;
+    let message = error.message;
+    if (typeof error.name != "number") {
```

Comment:
This is using the old logic for `handleServiceError`, use the updated one currently in dev